java/vector/src/main/java/org/apache/arrow/vector/ipc/ArrowMagic.java

```diff
@@ -27,9 +27,9 @@
 class ArrowMagic {
   private ArrowMagic(){}
 
-  private static final byte[] MAGIC = "ARROW1".getBytes(StandardCharsets.UTF_8);
+  static final byte[] MAGIC = "ARROW1".getBytes(StandardCharsets.UTF_8);
 
-  public static final int MAGIC_LENGTH = MAGIC.length;
+  static final int MAGIC_LENGTH = MAGIC.length;
```

Comment:
why make this package private?  This seems like an unnecessary API breakage?

Reply:
Sorry. I have reverted this change.